### DISCUSSION TITLE
change name of filament preheats

### DISCUSSION THREAD

**Discussion by jxer** [mriscoc/Ender3V2S1#165] [Q&A] [answered]
is there an easy way to change abs name and custom name to say petg or asa?

**Comment by mriscoc** [comment, marked as answer]
Yes, editing the labels in configuration.h file

https://github.com/mriscoc/Ender3V2S1/blob/fd215454c21c3b3afecca522e6141f52576aa4c7/Marlin/Configuration.h#L2013-L2028

**Reply by jxer** [reply]
This would require me to compile yes? no way to do this from the lcd?

**Reply by mriscoc** [reply]
Yes, you must to compile your own version.
